code/lighting/lighting.h

```diff
@@ -27,8 +27,15 @@
 #define LT_TUBE			2		// A tube light, like a fluorescent light
 #define LT_CONE			3		// A cone light, like a flood light
 
+enum class Light_Type : int {
```

Comment:
The enum values need to be explicitly specified here since you use the `int` value of this type above.